### PR TITLE
munkres

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

File: recipes/munkres/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  source_files:
+    - test/*
+  requires:
+    - nose
+  commands:
+    - nosetests
+
+about:
+  home: https://github.com/bmc/munkres
+  license: Apache v2
```
Comment (synapticarbors) [inline recipes/munkres/meta.yaml]: `license: Apache-2.0`

File: recipes/munkres/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
```
Comment (synapticarbors) [inline recipes/munkres/meta.yaml]: Could we throw in an `import` test just for good measure?

File: recipes/munkres/meta.yaml (hunk 1 above)
Comment (CJ-Wright) [reply recipes/munkres/meta.yaml]: One of these days we'll come up a list of these things (for at least the common ones).

Comment (synapticarbors) [reply recipes/munkres/meta.yaml]: I just go off of:

https://opensource.org/licenses/alphabetical

as mentioned in the example recipe:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L49-L51

**Pull request merged**